src/client.rs

```diff
@@ -1233,14 +1233,29 @@ where
     /// by this client.
     ///
     /// This limit is configured by the server peer by sending the
-    /// [`SETTINGS_MAX_CONCURRENT_STREAMS` parameter][settings] in a `SETTINGS`
-    /// frame. This method returns the currently acknowledged value recieved
-    /// from the remote.
+    /// [`SETTINGS_MAX_CONCURRENT_STREAMS` parameter][1] in a `SETTINGS` frame.
```

Comment:
I think this `[1]` change is half-way. The `[settings]: ....` below needs to be updated as well.

Reply:
oh whoops --- good catch!